### ISSUE TITLE
The output of CMake is very weirdly formatted

### ISSUE THREAD

**Issue by thatbakamono** [chalet-org/chalet#158]
```
-- CMake version: 3.23.1
                        -- The CXX compiler identification is MSVC 19.32.31329.0
                                                                                -- Detecting CXX compiler ABI info
                                                                                                                  -- Detecting CXX compiler ABI info - done
                                   -- Check for working CXX compiler: C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.32.31326/bin/HostX64/x64/cl.exe - skipped
                                                                -- Detecting CXX compile features
                                                                                                 -- Detecting CXX compile features - done
```

**Comment by rewrking** [comment]
Dang. I'll have a fix out of this by tomorrow.

**Comment by rewrking** [comment]
Fixed in [v0.4.1](https://github.com/chalet-org/chalet/releases/tag/v0.4.1)
